src/utils.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const gamesApi = axios.create({
+  baseURL: "https://nc-games-xkym.onrender.com/api",
```

Comment:
I like that you have set up a base URL for your API, and put it in a utils folder, super organised! 😄